src/Select.js

```diff
@@ -1081,7 +984,6 @@ const Select = createClass({
 		let className = classNames('Select', this.props.className, {
 			'Select--multi': this.props.multi,
 			'Select--single': !this.props.multi,
-			'is-clearable': this.props.clearable,
```

Comment:
Oops looks like they also removed this as well. 

This PR is starting to make me question if this conversion was done by hand or not 😱 .  @rgrocha how did you generate the code?